app/models/course.rb

```diff
@@ -261,6 +261,7 @@ def self.entry_requirement_options_without_nil_choice
   validate :validate_enrichment_publishable, on: :publish
   validate :validate_site_statuses_publishable, on: :publish
   validate :validate_provider_visa_sponsorship_publishable, on: :publish
+  validate :validate_all_sites_publishable, on: :publish, if: -> { recruitment_cycle.year.to_i >= 2022 }
```

Comment:
Can we take this approach and refactor it along the lines of `COURSE_2022_REQUIREMENTS_REQUIRED_FROM`

https://github.com/DFE-Digital/teacher-training-api/blob/master/app/models/course.rb#L58
```suggestion
  validate :validate_all_sites_publishable, on: :publish, if: -> { recruitment_cycle.year.to_i >= COURSE_2022_REQUIREMENTS_REQUIRED_FROM }
```

To inform future selves, otherwise we end up with pepper shotgun with `202X` over the years

Reply:
done 👍